config/database.yml

```diff
@@ -64,10 +64,10 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  url: <%= ENV['DATABASE_URL_TEST'] || ENV['DATABASE_URL'] || ''%>
-  database: PracticalDeveloper_test<%= ENV['TEST_ENV_NUMBER'] %>
+  url: <%= ENV.fetch("DATABASE_URL_TEST", "postgres:///Forem_test") %>
```

Comment:
We need to update our `travis.yml` file with these updated variables in order for the CI to pass.

Reply:
Awesome thanks for the info, I hadn't looked into why the build was failing.